content/en/docs/tasks/traffic-management/ingress/ingress-sni-passthrough/index.md

```diff
@@ -165,65 +175,150 @@ to hold the configuration of the NGINX server:
 
 ## Configure an ingress gateway
 
-1.  Define a `Gateway` with a `server` section for port 443. Note the `PASSTHROUGH` TLS mode which instructs
-    the gateway to pass the ingress traffic AS IS, without terminating TLS.
-
-    {{< text bash >}}
-    $ kubectl apply -f - <<EOF
-    apiVersion: networking.istio.io/v1alpha3
-    kind: Gateway
-    metadata:
-      name: mygateway
-    spec:
-      selector:
-        istio: ingressgateway # use istio default ingress gateway
-      servers:
-      - port:
+1.  Define a `Gateway` exposing port 443 with passthrough TLS mode. This instructs
+    the gateway to pass the ingress traffic "as is", without terminating TLS:
+
+{{< tabset category-name="config-api" >}}
+
+{{< tab name="Istio classic" category-value="istio-classic" >}}
+
+{{< text bash >}}
+$ kubectl apply -f - <<EOF
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: mygateway
+spec:
+  selector:
+    istio: ingressgateway # use istio default ingress gateway
+  servers:
+  - port:
+      number: 443
+      name: https
+      protocol: HTTPS
+    tls:
+      mode: PASSTHROUGH
+    hosts:
+    - nginx.example.com
+EOF
+{{< /text >}}
+
+{{< /tab >}}
+
+{{< tab name="Gateway API" category-value="gateway-api" >}}
+
+{{< text bash >}}
+$ kubectl apply -f - <<EOF
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: Gateway
+metadata:
+  name: mygateway
```

Comment:
do we want a namespace here? Or I guess we should do whatever we do in other places

Reply:
For now, I'm trying to keep the Gateway API and Istio versions of these docs as similar as possible. Generally the Istio versions don't include explicit namespaces, just for simplicity. I do agree we need to somehow figure out a good way to start educating users on the proper separation of gateway and route resources in different namespaces, but I don't think that docs like this one, which are just trying to demonstrate a specific function, is the right place.

Reply:
Fair enough